cluster/src/main/java/org/apache/iotdb/cluster/metadata/CMManager.java

```diff
@@ -1690,7 +1690,15 @@ public MNode getMNode(MNode deviceMNode, String measurementName) {
     ExecutorService pool =
         new ThreadPoolExecutor(
             THREAD_POOL_SIZE, THREAD_POOL_SIZE, 0, TimeUnit.SECONDS, new LinkedBlockingDeque<>());
-    List<PartitionGroup> globalGroups = metaGroupMember.getPartitionTable().getGlobalGroups();
+
+    List<PartitionGroup> globalGroups = new ArrayList<>();
+    if (IoTDBConstant.PATH_ROOT.equals(plan.getPath().getFullPath())) {
+      globalGroups = metaGroupMember.getPartitionTable().getGlobalGroups();
+    } else {
+      PartitionGroup partitionGroup =
+          metaGroupMember.getPartitionTable().partitionByPathTime(plan.getPath(), 0);
```

Comment:
show timeseries root.* is not right?

Reply:
Thanks @mychaow .  this is have a situation,  root.a.*
I  will alter this.